docs/user/openstack/install_upi.md

```diff
@@ -572,7 +594,19 @@ The main file needs to be uploaded to an HTTP(S) location the Bootstrap node wil
 
 You are free to choose any storage you want. For example:
 
-* Swift Object Storage with tempurl
+* Swift Object Storage
+  Create the `<container_name>` container and upload the `bootstrap.ign` file:
+  ```sh
+  $ swift upload <container_name> bootstrap.ign
+  ```
+  Make the container accessible:
+  ```sh
+  $ swift post <container_name> --read-acl ".r:*,.rlistings"
+  ```
+  Get the `StorageURL` from the output:
+  ```sh
+  $ swift stat -v
+  ```
```

Comment:
Why is this part of Kuryr documentation?

Reply:
This is more of an improvement from the previous UPI doc and it could be handy for the User.

Reply:
Fair with me.